File: PHP/Calculator/src/CalculatorBundle/Entity/Calculator.php
<?php
namespace CalculatorBundle\Entity;

class Calculator
{
    /**
     * @var float
     */
    private $leftOperand;

    /**
     * @var float
     */
    private $rightOperand;

    /**
     * @var string
     */
    private $operator;

    /**
     * Get left operand
     *
     * @return float
     */
    public function getLeftOperand()
    {
        return $this->leftOperand;
    }

    /**
     * Set left operand
     *
     * @param float $operand
     * @return Calculator
     */
    public function setLeftOperand($operand)
    {
        $this->leftOperand = $operand;
        return $this;
    }

    /**
     * Get Right operand
     *
     * @return float
     */
    public function getRightOperand()
    {
        return $this->rightOperand;
    }

    /**
     * Set Right operand
     *
     * @param float $operand
     * @return Calculator
     */
    public function setRightOperand($operand)
    {
        $this->rightOperand = $operand;
        return $this;
    }

    /**
     * Get Operator
     *
     * @return string
     */
    public function getOperator()
    {
        return $this->operator;
    }

        /**
     * Set Right operand
     *
     * @param string $operator
     * @return Calculator
     */
    public function setOperator($operand)
    {
        $this->operator = $operand;
        return $this;
    }

}

File: Java/JavaCalc/src/main/java/com/softuni/entity/Calculator.java
package com.softuni.entity;

public class Calculator {
    private Double leftOperand;
    private Double rightOperand;
    private String operator;

    public Calculator() {
    }

    public Calculator(Double leftOperand, Double rightOperand, String operator) {
        this.leftOperand = leftOperand;
        this.rightOperand = rightOperand;
        this.operator = operator;
    }

    public Double getLeftOperand() {
        return leftOperand;
    }

    public Double getRightOperand() {
        return rightOperand;
    }

    public String getOperator() {
        return operator;
    }

    public void setLeftOperand(Double leftOperand) {
        this.leftOperand = leftOperand;
    }

    public void setRightOperand(Double rightOperand) {
        this.rightOperand = rightOperand;
    }

    public void setOperator(String operator) {
        this.operator = operator;
    }

    public Double calculateResult() {
        double result;
        switch (this.operator) {
            case "+":
                result = this.leftOperand + this.rightOperand;
                break;
            case "-":
                result = this.leftOperand - this.rightOperand;
                break;
            case "*":
                result = this.leftOperand * this.rightOperand;
                break;
            case "/":
                result = this.leftOperand / this.rightOperand;
                break;
            case "%":
                result = this.leftOperand % this.rightOperand;
                break;
            case "^":
                result = Math.pow(this.leftOperand, this.rightOperand);
                break;
                default:
                    result = 0;
                    break;
        }
        return result;
    }
}
